backend/config/settings/site_specific.py

```diff
@@ -71,6 +67,12 @@ class MetagridFrontendSettings(BaseSettings):
         default="globus",
         description="Which authentication method to enable for user sign in on the frontend.",
     )
+
+    SEARCH_URL: str = Field(
+        description="The URL at which the ESG-Search api can be queried. A suitable endpoint will return XML in the browser.",
+        examples=["https://esgf-node.llnl.gov/esg-search/search"],
```

Comment:
Is it worth fixing these examples?  esgf-node.llnl.gov is just serving redirects to other services.

Reply:
Sure I can fix it, what would you prefer here?